spec/unit/default_spec.rb

```diff
@@ -70,6 +70,39 @@
       user: 'whats_fresh')
   end
   # rubocop:enable Metrics/LineLength
+
+  it 'installs and sets up gunicorn for Whats Fresh' do
+    expect(chef_run)
+      .to create_gunicorn_config('/opt/whats_fresh/gunicorn_config.py').with(
+        listen: '0.0.0.0:8888')
+  end
+
+  it 'installs and sets up supervisor for Whats Fresh' do
+    expect(chef_run)
+      .to enable_supervisor_service('whats_fresh').with(
+        command: '/opt/venv_whats_fresh/bin/gunicorn' \
+          ' whats_fresh.wsgi:application' \
+          ' -c /opt/whats_fresh/gunicorn_config.py',
+        autorestart: true,
+        directory: '/opt/whats_fresh')
+  end
+
+  it 'installs gunicorn to Whats Fresh virtualenv' do
```

Comment:
This is a generic project cookbook, but it makes specific references here and elsewhere to What's Fresh.  Is this intentional?